libsentrykube/tests/test_quickpatch.py

```diff
@@ -311,3 +362,103 @@ def test_validations(expected_message, arguments):
             TEST_PATCH,
             arguments,
         )
+
+
+@pytest.mark.parametrize(
+    "patch, resource, expected",
+    [
+        pytest.param(
+            [
+                {
+                    "path": "a/b/c",
+                    "value": TEST_NUM_REPLICAS,
+                }
+            ],
+            {},
+            {"a": {"b": {"c": TEST_NUM_REPLICAS}}},
+        ),  # Whole path is not present, should create
```

Comment:
Instead of using a comment, please provide an `id` argument to the `pytest.param`. That will make the id test appear in the output of pytest and make it clear which test has failed when they fail.https://github.com/getsentry/sentry-infra-tools/blob/main/config_builder/test_materializer.py#L74